test/reliability.jl

```diff
@@ -0,0 +1,42 @@
+using StatsBase
+using LinearAlgebra, Random, Test
+
+@testset "StatsBase.Reliability" begin
+    # basic vanilla test
+    cov_X = [10 6 6 6;
+             6 11 6 6;
+             6 6 12 6;
+             6 6 6 13]
+    reliability_X = crombach_alpha(cov_X)
+    @test reliability_X.alpha ≈ 0.8135593220338981
+    @test reliability_X.dropped[1].second ≈ 0.75
+    @test reliability_X.dropped[2].second ≈ 0.7605633802816901
+    @test reliability_X.dropped[3].second ≈ 0.7714285714285715
+    @test reliability_X.dropped[4].second ≈ 0.782608695652174
+
+    # testing Rational
+    cov_rational = 1 .// cov_X
+    reliability_rational = crombach_alpha(cov_X)
+    @test reliability_rational.alpha ≈ 0.8135593220338981
+    @test reliability_rational.dropped[1].second ≈ 0.75
+    @test reliability_rational.dropped[2].second ≈ 0.7605633802816901
+    @test reliability_rational.dropped[3].second ≈ 0.7714285714285715
+    @test reliability_rational.dropped[4].second ≈ 0.782608695652174
```

Comment:
```suggestion
    @test reliability_rational.alpha == 48 // 59
    @test reliability_rational.dropped[1].second == 3 // 4
    @test reliability_rational.dropped[2].second == 54 // 71
    @test reliability_rational.dropped[3].second == 27 // 35
    @test reliability_rational.dropped[4].second == 18 // 23
```

Reply:
this is a point of making `Rational` parametric - e.g. if someone wants to make a precise computation without rounding it is done as expected.